tests/linalg_test.py

```diff
@@ -671,8 +671,9 @@ def compare_orthogonal(q1, q2):
     self.assertTrue(np.all(
         norm(np.eye(k) - np.matmul(np.conj(T(lq)), lq)) < 5))
 
-    if not full_matrices and m >= n:
-        jtu.check_jvp(jnp.linalg.qr, partial(jvp, jnp.linalg.qr), (a,), atol=3e-3)
+    if (not full_matrices and m >= n) or (full_matrices and m == n):
```

Comment:
Shouldn't this be?

`if m  <= n or not full_matrices:`

Reply:
No, autodiff wil error for `m < n`